api/tests/certification/scoring/unit/domain/models/V3CertificationScoring_test.js

```diff
@@ -0,0 +1,77 @@
+import { V3CertificationScoring } from '../../../../../../src/certification/scoring/domain/models/V3CertificationScoring.js';
+import { domainBuilder, expect, sinon } from '../../../../../test-helper.js';
+
+describe('Unit | Certification | V3CertificationScoring', function () {
+  describe('#getCompetencesScore', function () {
+    it('should return the competences score', function () {
+      // Given
+      const estimatedLevel = 3;
+      const competence1Score = 1;
+      const competence2Score = 2;
+      const competenceForScoring1 = {
+        getCompetenceMark: sinon.stub(),
+      };
+      const competenceForScoring2 = {
+        getCompetenceMark: sinon.stub(),
+      };
+
+      const competence1Mark = domainBuilder.buildCompetenceMark({ score: competence1Score });
+      const competence2Mark = domainBuilder.buildCompetenceMark({ score: competence2Score });
+
+      competenceForScoring1.getCompetenceMark.withArgs(estimatedLevel).returns(competence1Mark);
+      competenceForScoring2.getCompetenceMark.withArgs(estimatedLevel).returns(competence2Mark);
+      const competencesForScoring = [competenceForScoring1, competenceForScoring2];
+      const certificationScoringConfiguration = {};
+
+      const v3CertificationScoring = new V3CertificationScoring({
+        competencesForScoring,
+        certificationScoringConfiguration,
+      });
+
+      // When
+      const competencesScore = v3CertificationScoring.getCompetencesScore(estimatedLevel);
+
+      // Then
+      expect(competencesScore).to.deep.equal([competence1Mark, competence2Mark]);
+    });
+  });
+
+  describe('.fromConfigurations', function () {
```

Comment:
Et-ce que le . c'est parce que static, ou une typo?
```suggestion
  describe('#fromConfigurations', function () {
```

Reply:
Typo.